pkg/v1/providers/config_default.yaml

```diff
@@ -553,6 +553,7 @@ AVI_USERNAME: ""
 AVI_PASSWORD: ""
 AVI_CLOUD_NAME:
 AVI_SERVICE_ENGINE_GROUP:
+AVI_CONTROLLER_VERSION: 20.1.3
```

Comment:
It seems odd to have a default version value here. Pretty much everything specific to version values are driven by the BOM. Can you explain why this case is different?

Reply:
The AVI controller is a VM deployed by customer. This is the version to tell AKO to use sdk with this version to talk to the AVI Controller.
Customer could change this value if they change their AVI controller VM's version

Reply:
Thanks